includes/debug-info.php

```diff
@@ -146,7 +146,7 @@ function add_debug_info( $info ) {
  *
  * @return array
  */
-function get_formatted_internal_connnections() {
+function get_formatted_internal_connections() {
```

Comment:
In case an external plugin is using these functions, it might be worth retaining the old names as wrappers for the new names and marking them as deprecated.

```php
get_formatted_internal_connnections() {
    ___deprecated_function( __FUNCTION__, 'x.x.x', __NAMESPACE__ . '\\get_formatted_internal_connections' );
    return get_formatted_internal_connections();
}
```

The same applies for the external getter below.

Reply:
Users of this plugin cannot interact with these two function.
- not possible to override
- not possible to filter

Reply:
I'm thinking of the case in which a plugin extending the functionality of this plugin calls these functions. 

As a simple example, a plugin that adds a WP CLI command to get the external connections:

```php
if ( ! class_exists( 'WP_CLI' ) ) {
	return;
}

/**
 * WP CLI Command to list external connections.
 */
if ( ! class_exists( 'WP_CLI' ) ) {
	return;
}

/**
 * WP CLI Command to list external connections.
 */
function external_connections() {
	// Typo in Distributor function name.
	$connections = Distributor\DebugInfo\get_formatted_external_connnections();
	$format_connections = [];
	foreach ( $connections as $name => $connection ) {
		$data = explode( "\n", $connection );

		$format_connection['Name'] = $name;

		foreach ( $data as $cell ) {
			$cell = explode( ': ', $cell, 2 );
			$format_connection[substr( $cell[0], 2 )] = $cell[1];
		}

		$format_connections[] = $format_connection;
	}

	WP_CLI\Utils\format_items( 'table', $format_connections, [ 'Name', 'URL', 'Version', 'Roles Allowed to Push' ] );
}
WP_CLI::add_command( 'dist', 'external_connections' );
```

Reply:
I see!

Reply:
Added.